### ISSUE TITLE
Wrong column names for queries with AS for h2 database 

### ISSUE THREAD

**Issue by szydan** [CraZySacX/node-jdbc#37]
I'm working with h2 database and for this query 
```
select investor.id as id1, investment.id as id2
from investor, investment_investor, investment
where 
investor.id = investment_investor.investorid
AND 
investment.id = investment_investor.investmentid
limit 1
```
In returned results the column names are ID and ID 

I looked at the 
https://github.com/CraZySacX/node-jdbc/blob/master/lib/jdbc.js#L98
where to get column name the **getColumnName** method is used 
var colname = rsmd.getColumnNameSync(i);

For h2 database however the **getColumnName** gives **ID** for both columns
If I switch the line to use  **.getColumnLabel** method instead
var colname = rsmd.getColumnLabelSync(i);
it gives the correct names for both columns which are **ID1** and **ID2**

According to javadoc from here
http://docs.oracle.com/javase/7/docs/api/java/sql/ResultSetMetaData.html#getColumnLabel(int)
```
Gets the designated column's suggested title for use in printouts and displays. The suggested title is usually specified by the SQL AS clause. If a SQL AS is not specified, the value returned from getColumnLabel will be the same as the value returned by the getColumnName method.                
```

So I think it would be better to use **getColumnLabel** method as it will give more correct column names in case where users use **AS** in their queries

**Comment by CraZySacX** [comment]
Fixed with 53d007b06b3faf580d9a07c5eca9d251a0d76781

**Comment by szydan** [comment]
Thanks :-)
